MultiplayerMod/Game/Mechanics/Printing/TelepadEvents.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection.Emit;
+using HarmonyLib;
+using MultiplayerMod.Core.Patch;
+using MultiplayerMod.Game.Mechanics.Minions;
+using MultiplayerMod.Multiplayer.Objects;
+using MultiplayerMod.Multiplayer.Objects.Reference;
+using UnityEngine;
+
+namespace MultiplayerMod.Game.Mechanics.Printing;
+
+[HarmonyPatch(typeof(Telepad))]
+public static class TelepadEvents {
+
+    public static event Action<AcceptDeliveryEventArgs>? AcceptDelivery;
+    public static event Action<GameObjectReference>? Reject;
```

Comment:
Why do use reference to GameObject instead of referencing KMonoBehaviour (Telepad) itself?